lib/room.rb

```diff
@@ -1,12 +1,13 @@
-class Rom
+class Room
```

Comment:
Keep a close eye on the interaction pattern - looks like we are missing two methods for the Room class - `is_painted?` and `paint`